### PR TITLE
CEM-1302-Dropdown

### PR DESCRIPTION
**Pull request by arkiis**
Add dropdown and dropdown item components. Add dropdown button as a prop to the dropdown so the user can choose what element triggers the dropdown. 

positoinElement

### CODE REVIEW

File: src/Containers/Dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+} from 'react';
+import styled from 'styled-components';
+import { transition,scroll } from '../../Utils/Mixins';
+import DropdownItem from './DropdownItem';
+
+type DropdownComponent<P = {}> = React.NamedExoticComponent<P> & {
+  Item: typeof DropdownItem
+}
+
+export interface IDropdownProps {
```
Comment (ralph-dev) [inline src/Containers/Dropdown/Dropdown.tsx:21]: Aren't you supposed to be extending the div props here?

**Review by ralph-dev — CHANGES_REQUESTED** [review]
Extend the props

File: src/Containers/Dropdown/Dropdown.tsx (hunk 1 above)
Comment (arkiis) [reply src/Containers/Dropdown/Dropdown.tsx]: Yeah I just added the props

**Review by ralph-dev — APPROVED** [review]
LGTM

**Review by ralph-dev — APPROVED** [review]
Placement should affect a fixed type of Left or Right instead of true or false becuase to extend in the future for up and down will require more changes now. It's still good to merge though!

**Pull request merged**